src/TriggerBinding/SqlTableChangeMonitor.cs

```diff
@@ -305,6 +305,9 @@ private async Task GetTableChangesAsync(SqlConnection connection, CancellationTo
 
                             getChangesDurationMs = commandSw.ElapsedMilliseconds;
                         }
+                        // Get the Lease Locked rows and log the number of rows.
+                        int leaseLockedRowCount = await this.GetLeaseLockedRowCount(connection, transaction);
+                        this._logger.LogDebug($"Executed GetChangesCommand in GetTableChangesAsync. {rows.Count} available changed rows ({leaseLockedRowCount} found with lease locks).");
```

Comment:
Let's only log this if `rows.Count > 0 || leaseLockedRowCount > 0` - otherwise we'll end up with a ton of messages where both values are 0 which isn't going to be much use.